### ISSUE TITLE
Hotkey for toggling alway-on-top

### ISSUE THREAD

**Issue by mmdmine** [Genymobile/scrcpy#2272]
It would be good if there was a hotkey to toggle always-on-top.

**Comment by PembaTamang** [comment]
Yes! this is what I was looking for. Please add this and thank you for this amazing tool!!!

**Comment by rom1v** [comment]
Just for information, what is your system/window manager?

On some environments (on Linux, at least Gnome, XFCE, probably KDE…), every window can be set "Always on top" by clicking on the title bar, then "Always on top".

**Comment by mmdmine** [comment]
@rom1v yeah, I've seen it. I'm using Windows and it doesn't have this option by default.

**Comment by rom1v** [comment]
I thought it would be easy, but in fact it seems SDL does not expose a function to change it. It's just exposed as a window flag when creating the window.
https://wiki.libsdl.org/SDL_WindowFlags

**Comment by PembaTamang** [comment]
So could you just add it as an option for now toggle can be done later.

**Comment by rom1v** [comment]
The option already exists.

https://github.com/Genymobile/scrcpy#always-on-top

**Comment by mmdmine** [comment]
Is it possible to recreate window? Or re-run scrcpy with `--always-on-top` flag with a hotkey.
Also, I know Windows has a [function](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setwindowpos) for making a window topmost, with window handle. I have no idea about mac os or xorg or wayland for other platforms.

**Comment by dumbasPL** [comment]
any progress on this? Should I add this and make a PR?

**Comment by rom1v** [comment]
> I thought it would be easy, but in fact it seems SDL does not expose a function to change it.

Now it does: https://discourse.libsdl.org/t/sdl-2-0-16-released/32471

> Added SDL_SetWindowAlwaysOnTop() to dynamically change the SDL_WINDOW_ALWAYS_ON_TOP flag for a window

> Should I add this and make a PR?

Yes, you could check that SDL is at least 2.0.16 and call that function to toggle.
